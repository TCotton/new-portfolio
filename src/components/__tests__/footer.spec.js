import React from 'react';
import renderer from 'react-test-renderer';

import Footer from '../header';

describe('Footer', () => {

	let component, instance;

	beforeEach( () => {
		component = renderer.create(
			<Footer />
		);
		instance = component.getInstance();
	});

	it('renders correctly', () => {
		const tree = renderer
			.create(<Footer />)
			.toJSON()
		expect(tree).toMatchSnapshot();
	})

	it('should call shouldComponentUpdate', () => {
		const shouldUpdate = instance.shouldComponentUpdate();
		expect(shouldUpdate).toBe(true);
	});
})
