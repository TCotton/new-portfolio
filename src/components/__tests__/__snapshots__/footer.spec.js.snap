// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders correctly 1`] = `
<header>
  <div>
    <h1>
      <a
        href="/"
      >
        
      </a>
    </h1>
  </div>
</header>
`;
